bundles/jsch-agentproxy-jsch/pom.xml

```diff
@@ -0,0 +1,94 @@
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>jclouds-karaf</artifactId>
+    <groupId>org.apache.jclouds</groupId>
+    <version>2.0.0-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.jclouds.karaf.bundles</groupId>
+  <artifactId>jsch-agentproxy-jsch</artifactId>
+  <name>jclouds :: Karaf :: JSCH Agentproxy Shaded Bundle</name>
+  <packaging>bundle</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.servicemix.bundles</groupId>
+      <artifactId>org.apache.servicemix.bundles.jsch-agentproxy-jsch</artifactId>
+      <version>${jsch.agentproxy.bundle.version}</version>
+    </dependency>
+  </dependencies>
+
+  <properties>
+    <import.packages>
+      ft.jzlib;resolution:=optional;version="[1.1,2)",
+      keypairgen;resolution:=optional,
+      signature;resolution:=optional,
+      userauth;resolution:=optional,
+      org.ietf.jgss;resolution:=optional,
+      javax.crypto,
+      javax.crypto.interfaces,
+      javax.crypto.spec,
+      javax.naming, 
+      javax.naming.directory, 
+      javax.net, 
+      javax.security.auth.x500
+    </import.packages>
+  </properties>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>2.3.7</version>
+        <configuration>
+          <instructions>
+            <Import-Package>${import.packages}</Import-Package>
+          </instructions>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>2.4.3</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>
```

Comment:
> This way we ignore it and we don't commit it to the SCM.

Ah, clear - this is **not** the default value, in other words. Thanks for clarifying!